Pearson-Advance/frontend-app-pearson-admin-portal
# PADV-527-V1 - Migrate pearson admin portal.

## Ticket

https://agile-jira.pearson.com/browse/PADV-527

## Description

We will deploy the VUE Admin Portal MFE: https://github.com/Pearson-Advance/frontend-app-pearson-admin-portal for the Olive migration. This PR upgrades dependencies to be aware with the platform.

## Changes Made

- [x] Upgrade dependencies in package.json
- [x] Upgrade package-lock.json
- [x] Upgrade .nvmrc to use `Node 18`
- [x] Update CI for olive version 

## How to test

- Up platform environment `make dev.up` with [course operations](https://github.com/Pearson-Advance/course_operations/tree/master/pearson_course_operation) Plugin.
- Go to the repo `frontend-app-pearson-admin-portal`.
- Run `npm install`.
- Run `npm start`.
- Log in as Global Admin user.
- Go to `localhost:8090` and verify the MFE.
